src/pages/Profile/index.js

```diff
@@ -23,90 +32,361 @@ import avatar from '../../assets/avatar.svg';
 class Profile extends React.Component {
   constructor(props) {
     super(props);
+    this.ref = {
+      name: React.createRef(),
+      email: React.createRef(),
+      position: React.createRef(),
+    };
     this.state = {
-      isChecked: true,
+      dataExpireValue: '6 months',
+      editView: false,
+      displayName: 'Anisha Swain',
+      emailVal: 'xyz.redhat@gmail.com',
+      positionVal: 'Associate Software Engineer',
+      accountType: 'Pbench',
+      accountStatus: 'Active',
+      color: '#0080005e',
     };
-  }
 
-  componentDidMount = () => {};
+    this.dataExpireOptions = [
+      { value: '2 months', label: '2 months', disabled: false },
+      { value: '6 months', label: '6 months', disabled: false },
+      { value: '1 year', label: '1 year', disabled: false },
+    ];
```

Comment:
Shouldnt one of these values be enabled serving as a default (ideally placeholder value)

Reply:
Yes, the default value is set in the `component state` which is grabbed from a mock user file(_which will be replaced with data from the server later_)

Reply:
Come to think of it ... why are we hardcoding the data retention intervals in the React component? Eventually the backend will need to be processing and validating these. I don't know how these came to be in the original design and UI mocks, but I wonder if these should be "enshrined" in Redux with a REST mock file like the user profile?